src/macos.rs

```diff
@@ -180,52 +135,6 @@ fn to_string<T: Into<OsString>>(str_in: T) -> Result<String, Error> {
     }
 }
 
-/// Uses the Drop trait to `release` the object held by `ptr`.
-#[repr(transparent)]
-struct OwnedObject {
```

Comment:
The rough equivalent to this in `objc2` is [`objc2::rc::Id`](https://docs.rs/objc2/0.5.1/objc2/rc/struct.Id.html).